cypress/support/constant.ts

```diff
@@ -0,0 +1,2 @@
+export const TWEET_COUNT = 10;
+export const CONTRIBUTORS_COUNT = 29;
```

Comment:
Length wouldn't be a great idea to put in TC

Reply:
Sure it is as we are checking if the expected elements are rendered or not, in the case of tweets we have hard coded the logic in the frontend side so we know we should be expecting the 10 tweets on the website, so I am checking the same here that all the requested tweets are available on the page or not!